### ISSUE TITLE
cohort_attribute table is deprecated

### ISSUE THREAD

**Issue by gowthamrao** [OHDSI/FeatureExtraction#87]
cohort_attribute table is deprecated in CDM 6, but is referenced in this package in 'Creating covariates using cohort attributes'

https://github.com/OHDSI/CommonDataModel/issues/212

Covariates using cohort attributes is an important and useful functionality. We just need a new home for the cohort_attribute table

**Comment by schuemie** [comment]
Yes, we should drop the reference to the CDM, and just clearly define the structure of the cohort_attribute table we need.
